src/controllers/Employees/deleteEmployee.ts

```diff
@@ -0,0 +1,13 @@
+import  { Request, Response } from 'express';
+import { empleados } from '../../fakedata/empleados.json';
+
+export default (req: Request, res: Response) =>{
+    const id: string = req.params.id;
+    const employeeFound: number = empleados.findIndex(employee => employee.id === id);
+
+    if(!employeeFound){
+        res.sendStatus(404);
+    }
```

Comment:
**antes y después** de un bloque IF, MAP, FOR, FILTER, REDUCE, etc, deja espacios